purchase_report_custom_hls/report/purchase_order_report_templates.xml

```diff
@@ -0,0 +1,38 @@
+<?xml version="1.0" ?>
+<odoo>
+    <template
+        id="report_purchaseorder_document_inherit_purchase_order"
+        inherit_id="purchase.report_purchaseorder_document"
+    >
+        <xpath expr="//div[@t-if='o.date_order']" position="before">
+            <div t-if="o.payment_term_id">
+                <strong>Payment Terms:</strong>
+                <br />
+                <span t-field="o.payment_term_id"/>
+            </div>
+        </xpath>
+        <xpath expr="//div[@class='page']" position="after">
+            <div t-if="o.company_id.rep_title">
+                <tr>
+                    <hr
+                        width="100%"
+                        style="border: none; border-bottom: 2px solid black;"
+                    />
+                    <td width="100%">
```

Comment:
I suppose the width attribute is not needed.

Wasn't sure if the `td` tag was necessary (I just don't know).

Reply:
I edited div class.
If don't define width with block element, `hr` width will be same as 100vw.

Reply:
I deleted it.
 Thank you.